coderd/database/migrations/000230_notifications_fix_username.down.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE notification_templates
+SET
+    actions = REPLACE(actions::text, '@{{.UserUsername}}', '@{{.UserName}}')::jsonb;
```

Comment:
We can't replace this everywhere; when we greet the user we should use `UserName`, but when we reference their account we should use `UserUsername`.

Reply:
It replaces values only in the actions column, right?

Reply:
🤦 missed that 👍